client/my-sites/theme/main.jsx

```diff
@@ -348,46 +348,58 @@ const ThemeSheet = React.createClass( {
 		const analyticsPageTitle = `Themes > Details Sheet${ section ? ' > ' + titlecase( section ) : '' }${ siteID ? ' > Site' : '' }`;
 
 		const { name: themeName, description } = this.props;
-		const title = i18n.translate( '%(themeName)s Theme', {
+		const title = themeName && i18n.translate( '%(themeName)s Theme', {
```

Comment:
What was the reason for the `themeName &&` change? This won't prevent the `<DocumentHead />` from dispatching the title, will it?

Per: https://github.com/Automattic/wp-calypso/blob/9923003f10d4e89f120bab7d9bce2bc49dcb42fa/client/components/data/document-head/index.jsx#L29-L31